server/utils_test.go

```diff
@@ -67,12 +67,12 @@ func TestParseOwnerAndRepo(t *testing.T) {
 		{Full: "https://github.com/mattermost/mattermost-server", BaseURL: "", ExpectedOwner: "mattermost", ExpectedRepo: "mattermost-server"},
 		{Full: "https://github.com/mattermost/mattermost-server", BaseURL: "https://github.com/", ExpectedOwner: "mattermost", ExpectedRepo: "mattermost-server"},
 		{Full: "", BaseURL: "", ExpectedOwner: "", ExpectedRepo: ""},
-		{Full: "mattermost/mattermost/invalid_repo_url", BaseURL: "", ExpectedOwner: "", ExpectedRepo: ""},
-		{Full: "https://github.com/mattermost/mattermost/invalid_repo_url", BaseURL: "", ExpectedOwner: "", ExpectedRepo: ""},
+		{Full: "mattermost/mattermost/invalid_repo_url", BaseURL: "", ExpectedOwner: "mattermost", ExpectedRepo: "mattermost"},
```

Comment:
0/5 would be awesome to add tests for ... shortening

Reply:
Are you suggesting something like this:
```
{Full: "mattermost/...server", BaseURL: "", ExpectedOwner: "mattermost", ExpectedRepo: "...server"},
```

Reply:
Are you suggesting something like this:
```
{Full: "mattermost/...server", BaseURL: "", ExpectedOwner: "mattermost", ExpectedRepo: "...server"},
```

Reply:
Yes, exactly